lib/srv/authhandlers.go

```diff
@@ -87,6 +92,15 @@ type AuthHandlerConfig struct {
 	// for TLS certificate verification.
 	// Defaults to real clock if unspecified
 	Clock clockwork.Clock
+
+	// AuthorizeWithRBAC specifies the function to preform user
+	// authorization and RBAC checks in tests. It can safely be unset
+	// otherwise.
+	AuthorizeWithRBAC func(cert *ssh.Certificate, clusterName string, node types.Server, teleportUser string, osUser string) error
```

Comment:
Since we're embedding the interface for RBAC, why do we still need these function pointers here?

Reply:
Oops, removed